app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```

Comment:
Can you also add validation to make sure that the user has a username? After doing this you're going to want to look into devise custom views. https://github.com/heartcombo/devise#configuring-views Since you will be validating that the user will have a username you will also have to update the form to let them submit one.

Reply:
Third commit has username validations.